index.js

```diff
@@ -44,10 +44,12 @@ module.exports = function configParser(yaml, templateFactory) {
         .then(phaseGeneratePermutations)
         // Output in the right format
         .then(doc => ({
+            annotations: Hoek.reach(doc, 'annotations', { default: {} }),
```

Comment:
For the pipeline level annotation, I don't think we can do it now without sequelize 4.x since we cannot create a new field `annotations`. :(

When it goes through the pipeline.sync, it will only update the workflow for the pipeline.
https://github.com/screwdriver-cd/models/blob/master/lib/pipeline.js#L251

It probably won't break anything but it's also not doing anything.. we won't be able to see it from `/pipelines/id`.